robolectric/src/test/java/org/robolectric/shadows/ShadowColorTest.java

```diff
@@ -27,8 +27,6 @@ public void testParseColor() throws Exception {
     assertThat(Color.parseColor("#00000000")).isEqualTo(0);
 
     assertThat(Color.parseColor("#ffaabbcc")).isEqualTo(-5588020);
-    assertThat(Color.parseColor("#fabc")).isEqualTo(-5588020);
-    assertThat(Color.parseColor("#abc")).isEqualTo(-5588020);
```

Comment:
What's the motivation for removing the assertions? It would be nice to have at least one assertion for each string format thats supported. I could also be convinced that the tests should move to `ResourceHelperTest` (which doesn't exist).